docs/reference/elasticsearch-migration-2021-02-15.md

```diff
@@ -2,14 +2,29 @@
 
 Release `2021-02-15` of `wire-server` requires creating a new ElasticSearch index for `brig` _before_ deploying the release. Without this new index the user search in TeamSettings will be defunct.
 
-The index that brig is using, is defined at brig's config at `elasticsearch.index`. This config value of the previous deployment is referred to as `<OLD_INDEX>` in the following.
+The index that brig is using, is defined in the `brig` chart's config at `elasticsearch.index`. This config value of the previous deployment is referred to as `<OLD_INDEX>` in the following.
 
 These following instructions describe how to:
 
 1. Create the new index `<NEW_INDEX>` (you can choose any name that is different from `<OLD_INDEX>`)
 2. Populate `<NEW_INDEX>` with brig's data.
 3. Configure brig to use the new index.
 
+Note: The following steps require downtime of brig. If downtime is not acceptable then please skip to section [Create the new index without downtime](#create-the-new-index-without-downtime)
```

Comment:
Why would this cause downtime of brig?  I think `elasticsearch-index` gets run and then and only then `brig` will get a rolling release.  (How currently the chart is set up)

Reply:
Ah yes, this note is misleading. There is no downtime. If you don't shut down brig before the migration then you risk having missing user updates of brig in ES.

Reply:
I think there are 2 ways I could update these instructions:

1) Prepend a step "Shut down brig."
OR
2) Remove the note about downtime and append a step to the instructions
    "Reindex brig's data again by ... This makes sure that the ES index and brig's data are fully in sync"

Which one would you prefer? Or do you see any alternative?